analysis_functions/dns_spoofing_detection.py

```diff
@@ -0,0 +1,46 @@
+from time       import sleep
+from queue      import Queue
+from socket     import inet_ntoa
+from translator import translator_get_arp_cache, translator_update_arp_cache
+from helpers    import get_keep_running_val, add_colons_to_mac
+from binascii   import hexlify
+import                 dpkt
```

Comment:
Нигде не используются.
```suggestion
from queue      import Queue
from helpers    import get_keep_running_val
import                 dpkt
```